Tools/SandboxTest.ps1

```diff
@@ -0,0 +1,125 @@
+# Parse Arguments
+
+Param(
+  [Parameter(Mandatory, HelpMessage = "The path for the Manifest.")]
+  [String] $Manifest
+)
+
+if (-not (Test-Path -Path $Manifest -PathType Leaf)) {
+  throw 'The Manifest file does not exist.'
+}
+
+# Validate manifest file
+# We can't rely on status code until https://github.com/microsoft/winget-cli/issues/312 is solved
+$validationResult = winget.exe validate $Manifest
+if ($validationResult -like '*Manifest validation failed.*') {
+  throw 'Manifest validation failed.'
+}
+
+# Check if Windows Sandbox is enabled
+
+if (-Not (Test-Path "$env:windir\System32\WindowsSandbox.exe")) {
+  Write-Error -Category NotInstalled -Message @'
+Windows Sandbox does not seem to be available. Check the following URL for prerequisites and further details:    
+https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview
+  
+You can run the following command in an elevated PowerShell for enabling Windows Sandbox:
+Enable-WindowsOptionalFeature -Online -FeatureName 'Containers-DisposableClientVM'
+'@ -ErrorAction Stop
+}
+
+# Set dependencies

```

Comment:
Just a thought - with these dependencies hard-coded, we will need to update the script each time we release. Have you thought about downloading the files from the latest api? (https://api.github.com/repos/microsoft/winget-cli/releases/latest). Or you could use it as a prompt to notify a user of the script that a new client is available.

Reply:
Yes, I thought about that. I was wondering to wait for the second release of `winget` to catch the pattern (I don't know if the file name is going to change or not). But that's good, I'll implement it now.

Reply:
I believe @KevinLaMS and @denelon will primarily be taking point on deciding when to release an update (please correct me if I am wrong). Unless there is a good reason for having the _8wekyb3d8bbwe, we could probably remove that from the future releases and just go with Microsoft.DestkopAppInstaller.appxbundle

Reply:
If `8wekyb3d8bbwe` is immutable, this wouldn't be a problem.

Reply:
That is the Microsoft PublisherId. It's the same for all apps published by Microsoft and should not change.

Reply:
GitHub Releases API doesn't provide a hash for the assets, and without a hash, we can't verify if the existing binary is valid or not for triggering the download again. I think we should publish the hashes in some other place, maybe in the description of the release for this to work.

Reply:
@zachcarp any opinion?

Reply:
The hash does present a problem. I see 3 alternatives.

1. Leave it as is for now, and revisit this after we figure out the other major issues. This lets us get this merged sooner.
2. Publish the hash somewhere. Your idea of putting it in the description works, but we will need to wait for another release for that to happen.
3. While not ideal, it at least deserves a mention. You can download the package without a hash, and hash it in your script. This allows you to move forward now, and guarantee fidelity when loading to the sandbox, but does nothing about ensuring fidelity when consuming from GitHub.

With those in mind, I am inclined to go with the first one for now. When we get another release, we can look at revisiting this. That also lets us focus on the dependency issue now.

Reply:
I also agree with option one for now.